packages/account-v2/src/containers/PersonalDetailsForm/PersonalDetailsForm.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { Form, Formik } from 'formik';
+import {
+    useActiveTradingAccount,
+    useGetAccountStatus,
+    useIsEuRegion,
+    useResidenceList,
+    useSettings,
+    useStatesList,
+} from '@deriv/api-v2';
+import { Checkbox, Text } from '@deriv-com/ui';
+import { FormSubHeader } from '../../components/FormSubHeader';
+import { useCurrentLandingCompany } from '../../hooks/useCurrentLandingCompany';
+import { AddressFields } from '../../modules/AddressFields';
+import { PersonalDetails } from './PersonalDetails';
+import { SupportProfessionalClient } from './SupportProfessionalClient';
+import { TaxInformation } from './TaxInformation';
+import { getPersonalDetailsInitialValues, getPersonalDetailsValidationSchema } from './validation';
+
+export const PersonalDetailsForm = () => {
+    const { data: activeAccount } = useActiveTradingAccount();
+    const { data: accountSettings } = useSettings();
+    const { data: accountStatus } = useGetAccountStatus();
+    const { data: currentLandingCompany } = useCurrentLandingCompany();
+    const { data: residenceList } = useResidenceList();
+    const { data: statesLists } = useStatesList(accountSettings?.country_code ?? '');
+    const { data: isEu } = useIsEuRegion();
+
+    const isSocialSignup = accountStatus?.status?.includes('social_signup');
+
+    const isVirtual = activeAccount?.is_virtual;
+
+    const isSupportProfessionalClient =
+        currentLandingCompany &&
+        typeof currentLandingCompany === 'object' &&
+        'support_professional_client' in currentLandingCompany &&
+        !!currentLandingCompany?.support_professional_client;
+
+    const initialValues = getPersonalDetailsInitialValues(accountSettings, residenceList, statesLists, isSocialSignup);
+
+    const validationSchema = getPersonalDetailsValidationSchema(isEu);
+    return (
+        <Formik
+            enableReinitialize
+            initialValues={initialValues}
+            /*eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
**NIT**: Better to Go with a TODO rather than eslint disable. With TODOs we can track pending work in Sonarcloud as well